src/common/components/nav/nav.component.js

```diff
@@ -88,6 +90,12 @@ class NavController{
     // Register signedOut event on child scope
     // this basically sets the listener at a lower priority, allowing $rootScope listeners first chance to respond
     this.$rootScope.$new(true, this.$rootScope).$on(SignOutEvent, (event) => this.signedOut(event) );
+
+    try {
+      angular.module('common');
+    } catch(err) {
+      this.analyticsFactory.pageLoaded(true); // Only call pageLoaded when common module doesn't exist, indicating that only the nav bundle is running and we aren't on the give site
```

Comment:
I could do your `hostname.indexOf('give')` check here if that sounds safer/better.